docs/additional_files/system_requirements.md

```diff
@@ -0,0 +1,80 @@
+
+SnappyData turns Apache Spark into a mission critical, elastic scalable in-memory data store. This allows users to run Spark workloads and classic database workloads on SnappyData.
+
+**Memory**: SnappyData can run well with anywhere from 8GB of memory to hundreds of GB of memory. While exact memory requirements depend on the end user application, we recommend allocating no more than 75% of the memory to SnappyData. We recommend using a machine with at least 8GB of RAM when working with SnappyData
+
+**CPU Cores**: SnappyData is a highly multi-threaded system and is able to take advantage of CPU cores to deliver higher throughput. It has been tested with multi-core multi-CPU machines. We recommend using machines with at least 16 cores when working with SnappyData. The degree of parallelism you can achieve with SnappyData directly depends on the number of cores and higher core machines will perform better than lower core machines.
+
+**Network**: SnappyData is a clustered scale out in memory data store and both jobs and queries use the network extensively to complete their job. Since data is mostly available in memory, queries and jobs typically get CPU and/or network bound. We recommend running SnappyData on at least a 1GB network for testing and use a 10GB network for production scenarios.
+
+**Disk**: SnappyData overflows data to local disk files and tables can be configured for persistence. We recommend using flash storage for optimal performance for SnappyData shared nothing persistence. Data can be saved out to stores like HDFS and S3 using SnappyData DataFrame APIs
+
+
+## Operating Systems Supported
+
+| Operating System| Version |
+|--------|--------|
+|Red Hat Enterprise Linux|  **Amogh**      |
+|Ubuntu|        |
+|CentOS|        |
+
+
+## Host Machine Requirements
+Requirements for each host:
+
+* A supported [Java SE installation](http://www.oracle.com/technetwork/java/javase/downloads/jdk-6u26-download-400750.html).
+
+* File system that supports long file names.
+
+* Adequate per-user quota of file handles (ulimit for Linux). Ensure that you set a high enough ulimit (up to the hard limit of 81,920) for your system.
+
+* TCP/IP.
+
+* System clock set to the correct time.
+
+* For each Linux host, the hostname and host files must be properly configured. See the system manpages for hostname and hosts.
+
+* For each Linux host, configure the swap volume to be the same size as the physical RAM installed in the computer.
+
+* Time synchronization service such as Network Time Protocol (NTP).
+
+!!! Note: 
+	* For troubleshooting, you must run a time synchronization service on all hosts. Synchronized time stamps allow you to merge log messages from different hosts, for an accurate chronological history of a distributed run.
+	* If you deploy RowStore to a virtualized host, see also [Running RowStore in Virtualized Environments](http://rowstore.docs.snappydata.io/docs/manage_guide/tuning-vm.html#concept_20E2D15D2F65442CAA0B04E9EA144F1D).
```

Comment:
Why is this specific to the Row store? Probably applies to the product in general.